pthread/pthread.c

```diff
@@ -51,6 +55,38 @@ static const pthread_attr_t pthread_attr_default = {
 };
 
 
+static void __pthread_ctx_get(pthread_ctx *ctx)
+{
+	++ctx->refcount;
+}
+
+
+static void __pthread_ctx_put(pthread_ctx *ctx)
```

Comment:
Nitpick - we use single '_' to mark unlocked function. Or is there some reason for two '_'s?
```suggestion
static void _pthread_ctx_get(pthread_ctx *ctx)
{
	++ctx->refcount;
}


static void _pthread_ctx_put(pthread_ctx *ctx)
```